engine/src/main/java/org/destinationsol/game/SaveManager.java

```diff
@@ -46,10 +46,11 @@
 import java.io.UnsupportedEncodingException;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Optional;
 
 public class SaveManager {
 
-    private static Logger logger = LoggerFactory.getLogger(SaveManager.class);
+    private static final Logger logger = LoggerFactory.getLogger(SaveManager.class);
```

Comment:
AllCaps? Or are we not running that convention? Been a while...

Reply:
Good question... I think we are in some places but not others...